src/options.html

```diff
@@ -142,13 +151,38 @@ <h3>Settings Page</h3>
                            </div>
                      </div>
                 </div>
-                <div style="text-align: center;">
-                    <button class="btn btn-danger" id="restore">Restore Defaults</button>
+            </div>
+            <div class="row">
+                <div class="col-sm-12">
+                    <div class="row">
+                        <div class="col-sm-12 sub-heading">
+                            <span class="glyphicon glyphicon-volume-up"></span>
+                            <span>Susi's Voice</span>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-sm-4">
+                            <div class="btn-group" style="color: #0084ff;" id="getVoice">
```

Comment:
you can add the css into the .css file instead of using the inline css inside the html file.
thank you :smiley:

Reply:
and maybe a change from inline css to the .css file would be suitable so that the html file only contains the html tags and not inline css.
@ms1398 reviews ?

Reply:
@gabru-md  Ok